bikeshed/boilerplate/webgl/status.include

```diff
@@ -0,0 +1,3 @@
+<p>This document is an editor's draft. Do not cite this document as other than work in progress.</p>
+
+<p>[STATUSTEXT]</p>
```

Comment:
Does this status.include properly handle the case where the specification is promoted from draft to official?

Reply:
@kenrussell , I suppose that you want to format multiple Khronos WebGL related specifications via bikeshed, and these specifications are at different status (working draft, editor draft, ratified/offical, etc). 

Bikeshed can meet this requirement. We only need to submit another pr to add multiple status.include files. For example, rename this one to status-ED.include for editor's draft, and add status-WD.status for working status, etc.

In addtion, we need to add all other status like working draft and ratified into bikeshed/config/status.py. You can see that I have already added khronos/ED for editor's draft in that file.  

You see, it's quite easy and straightforward. And I verified the method above on my local machine.